data/112/606/087/7/1126060877.geojson

```diff
@@ -5,7 +5,7 @@
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
     "geom:area":0.0,
-    "geom:area_square_m":0.0,
+    "geom:area_square_m":0,
```

Comment:
hmmm... exportify tool?

Reply:
this is the command I'm using to run `exportify` in docker:
https://github.com/pelias/wof/blob/master/whosonfirst/exportify.js

Reply:
I'm a bit confuddled by this, why was `geom:area_square_m` converted to `0` but the line above `geom:area` remained `0.0` 😕